### ISSUE TITLE
Types of color blindness incorrectly ordered in the accessibility add-on

### ISSUE THREAD

**Issue by maya** [storybookjs/storybook#14767]
**Describe the bug**
In the accessibility add-on, the color blindness listings should be ordered by most to least common types of color blindness.

Use the order in this list: https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/color-blindness/types-color-blindness

Importantly, **Deuteranomaly should be first as it's _by far_ the most common**.

**Note**: it's fine to leave blurred vision first (as it's also super common) and this issue is just about the color blindness.

**To Reproduce**
Make sure the accessibility add-on is on, go to the storybook library, click on the accessibility button, view the dropdown.

**System**
N/A

**Additional context**
<img width="268" alt="Screen Shot 2021-04-29 at 6 03 38 PM" src="https://user-images.githubusercontent.com/5249443/116636171-8377b580-a915-11eb-955c-881e1f6ca82b.png">

**Comment by shilman** [comment]
Ermahgerd!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.3.0-alpha.17 containing PR #14768 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```

Closing this issue. Please re-open if you think there's still more to do.

**Comment by maya** [comment]
Awesome, thanks! The other improvement I could see is adding categories like this but that is probably not as straightforward as these changes.

![707CAC2F-4CAC-4C61-808D-98DDA1B101DB](https://user-images.githubusercontent.com/5249443/116739463-91c2e180-a9a8-11eb-9942-29a4dc31b1d6.jpeg)
